haystack/backends/elasticsearch_backend.py

```diff
@@ -186,8 +186,11 @@ def update(self, index, iterable, commit=True):
 
         bulk_index(self.conn, prepped_docs, index=self.index_name, doc_type='modelresult')
 
-        if commit:
-            self.conn.indices.refresh(index=self.index_name)
+        # We are explicitly eliminating the call to self.conn.indices.refresh().
```

Comment:
Everything i have read leads me to believe that the refresh happens once a second, not once a minute, so this may be a moot point.  

Let's say for the sake of argument though that the refresh interval is 1 minute.  During that 1 minute, window any changes sent up after the last refresh won't be reflected in queries.  

We should vet taht changes get reflected in queries, but i think this is fine